.babelrc

```diff
@@ -1,6 +1,5 @@
 {
   "presets": [
-    "es3",
     ["es2015", {
       "loose": true
```

Comment:
don't we only do loose for IE < 11 as well?

Reply:
we don't need loose mode but the filesize output from loose mode is smaller compared to regular.